std/parallelism.d

```diff
@@ -2633,7 +2633,7 @@ public:
 
             // Hack to take the address of a nested function w/o
             // making a closure.
-            static auto scopedAddress(D)(scope D del)
+            static auto scopedAddress(D)(D del)
```

Comment:
As per the comment above, this is common pattern that various projects use (e.g. vibe.d) to ensure that the clojure is not allocated with the GC. AFAIK, there's no other way to achieve this, because you can't (and it's not practical to) require callers to be `@nogc`, because in general they're not under your control.
This particular case could be worked around, but it still a breaking change for user code.